package.json

```diff
@@ -74,6 +74,7 @@
     "enzyme-adapter-react-16": "^1.15.6",
     "enzyme-to-json": "^3.6.2",
     "eslint": "^8.5.0",
+    "eslint-plugin-import": "^2.26.0",
```

Comment:
I don’t think this addition is needed, since this package is already defined as a peer dependency of `@wagtail/eslint-config-wagtail`. I believe we could just `npm install --save-dev @wagtail/eslint-config-wagtail@latest` and that will update this peer dependency’s version.

Reply:
Ok thank you - I must have been doing this wrong.